dist/css/bootstrap-grid.css

```diff
@@ -51,10 +51,8 @@ html {
   margin-left: auto;
 }
 .row {
-  display: -ms-flexbox;
   display: flex;
-  -ms-flex-wrap: wrap;
-      flex-wrap: wrap;
+  flex-wrap: wrap;
```

Comment:
Yes, that's entirely because of the browser settings for Autoprefixer. Although that would represent support as far back as IE10, which is barely used. However it's what Bootstrap v4 has. See: https://github.com/twbs/bootstrap/blob/v4-dev/.browserslistrc

Really you should just use the same browser list file if you want the same output.

Another good thing to do would be to import Sass Bootstrap as a dependency and run a diff tool on the generated CSS from Sass and from this port, so you could verify it.

Reply:
Btw, just another word on IE10 -> It has a smaller user base than IE8 / IE9. So to support it and not support IE8 / IE9 hardly makes any sense, so I dunno, I feel like it's a counter-intuitive setting.